Add load balancing implementation for Apache in documentation (#12159)
bokeh/bokeh: sphinx/source/docs/user_guide/server.rst

    The Bokeh server is horizontally scalable. To add more capacity, you
    can run more servers behind a load balancer.

> You can run as many Bokeh servers as you need. In the following, let us
> suppose that you have three Bokeh servers running on separate ports:

Reviewer: Following https://docs.bokeh.org/en/latest/docs/dev_guide/documentation.html#principles-of-good-style, I'd replace the first person 'us' with something else:

```suggestion
You can run as many Bokeh servers as you need. The following examples
are based on a setup with three Bokeh servers running on three different
ports:
```

.. code-block:: sh

    bokeh serve myapp.py --port 5100
    bokeh serve myapp.py --port 5101
    bokeh serve myapp.py --port 5102

> The sections below propose basic configurations, but please also refer to the
> `Nginx load balancer documentation`_ or the `Apache proxy balancer module
> documentation`_. For instance, there are different strategies available for
> choosing how incoming connections are distributed among instances.

Reviewer: For clarity, I'd turn the first sentence into two sentences:
```suggestion
The sections below propose basic configurations based on this setup. See the
`Nginx load balancer documentation`_ or the `Apache proxy balancer module
documentation`_ for more detailed information. For instance, there are
different strategies available to define how incoming connections are
distributed among server instances.
```

    }

Apache
''''''

> First make sure you have enabled the ``proxy_balancer`` and ``rewrite``

Reviewer: ```suggestion
First, make sure you have enabled the ``proxy_balancer`` and ``rewrite``
```

> modules. Then you need to add balancers for both http and websocket protocols.

Reviewer: ```suggestion
modules. 

Add balancers for both http and websocket protocols:
```

.. code-block :: apache

    <Proxy "balancer://myapp_http">
        BalancerMember "http://127.0.0.1:5100/myapp"
        BalancerMember "http://127.0.0.1:5101/myapp"
        BalancerMember "http://127.0.0.1:5102/myapp"
        ProxySet lbmethod=byrequests
    </Proxy>

    <Proxy "balancer://myapp_ws">
        BalancerMember "ws://127.0.0.1:5100/myapp"
        BalancerMember "ws://127.0.0.1:5101/myapp"
        BalancerMember "ws://127.0.0.1:5102/myapp"
        ProxySet lbmethod=byrequests
    </Proxy>

> Finally you can proxy connections to the two balancers:

Reviewer: ```suggestion
Finally, you can proxy connections to the two balancers:
```
